CollapsibleExample/MHTableViewCell.h

```diff
@@ -0,0 +1,51 @@
+//
+//  MHTableViewCell.h
+//  CollapsibleExample
+//
+//  Created by Lizzy Randall on 5/18/15.
+//  Copyright (c) 2015 Lizzy Randall. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+#import <Foundation/Foundation.h>
+
+@interface MHTableViewCell : UITableViewCell
+
+//Describes type of cell for interactions
+//Good to check when cell is selected for view controllers/etc. to handle
+//the interaction respectively
+typedef NS_ENUM(NSUInteger, CRUCellViewInteractionType){
+    CRUCellViewInteractionHeader, //for arrow image/look of header
+    CRUCellViewInteractionCheckToggle, //normal row that toggles a checkmark
+    CRUCellViewInteractionCheckList, //modal with uitableview to check multiple types
+    CRUCellViewInteractionPicker, //modal with uipicker to pick one
+    CRUCellViewInteractionTextBox //modal with textbox for user to enter manually
+};
+
+//calls super first then sets defaults
+- (instancetype) initWithStyle:(UITableViewCellStyle)style reuseIdentifier:(NSString *)reuseIdentifier;
+
+- (void)setMHCellDefaultsWithAccessory:(UITableViewCellAccessoryType)accessory style:(UITableViewCellSelectionStyle)style labelColor:(UIColor*)color accessoryView:(UIView*)view;
```

Comment:
I am loving your naming. It is on point. This is the only thing I could pick up and it is a confusing part of objc programming. Normally you wouldn't use MH prefix in your code. The prefix normally only occurs in naming a class. When referencing an instance of a class with the prefix you don't use it. So in this case the name would be:
```
- (void)setCellDefaultsWithAccessory:(UITableViewCellAccessoryType)accessory style:(UITableViewCellSelectionStyle)style labelColor:(UIColor*)color accessoryView:(UIView*)view;
```
It is a confusing way to do namespacing. Fortunately they don't use prefixes in swift, they actually implemented a proper namespacing system.